README.md

```diff
@@ -1,27 +1,61 @@
-### Run go-ipfs on docker
+# Orbit Chat
 
-Start the node:
+[![](https://img.shields.io/badge/freenode-%23orbitdb-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23orbitdb)
 
-    docker run -d \
```

Comment:
Did we get rid of all of the docker?

Reply:
What @JaniAnttonen means is that we never actually used Docker or had a Dockerfile in here. The old readme had just some of my own notes left over.

Reply:
Sweet. Just checking!